### PR TITLE
Add env var support

### PR DESCRIPTION
**Pull request by andresgomezfrr**
Process env var from task context

### CODE REVIEW

File: pkg/flink/types.go [hunk 1]
```diff
@@ -63,6 +63,15 @@ func GetDefaultLabels(taskCtx pluginsCore.TaskExecutionMetadata) Labels {
...
 
+type EnvironmentVariables map[string]string
+
+func GetDefaultEnvironmentVariables(taskCtx pluginsCore.TaskExecutionMetadata) EnvironmentVariables {
+	return utils.UnionMaps(
+		config.GetK8sPluginConfig().DefaultEnvVars,
+		utils.CopyMap(taskCtx.GetEnvironmentVariables()),
+	)
+}
+
```
Comment (regadas) [inline pkg/flink/types.go:74]: Hi @andresgomezfrr is this still relevant? can you expand on why is this needed?

File: pkg/flink/handler.go [hunk 2]
```diff
@@ -92,11 +93,12 @@ func NewFlinkTaskContext(ctx context.Context, taskCtx FlinkTaskExecContext) (*Fl
...
-		ClusterName: cn,
-		Namespace:   taskMetadata.GetNamespace(),
-		Annotations: GetDefaultAnnotations(taskMetadata),
-		Labels:      GetDefaultLabels(taskMetadata),
-		Job:         job,
+		ClusterName:          cn,
+		Namespace:            taskMetadata.GetNamespace(),
+		Annotations:          GetDefaultAnnotations(taskMetadata),
+		Labels:               GetDefaultLabels(taskMetadata),
+		EnvironmentVariables: GetDefaultEnvironmentVariables(taskMetadata),
```
Comment (regadas) [inline pkg/flink/handler.go:100]: 🎨 we could use a shorter name here `EnvVars` and have `GetDefaultEnvVars`

**Review by regadas — COMMENTED** [review]
I guess this can be useful in the future. @andresgomezfrr can you rebase this plz? Thx

File: pkg/flink/handler.go [hunk 3]
```diff
@@ -92,11 +93,12 @@ func NewFlinkTaskContext(ctx context.Context, taskCtx FlinkTaskExecContext) (*Fl
...
-		ClusterName: cn,
-		Namespace:   taskMetadata.GetNamespace(),
-		Annotations: GetDefaultAnnotations(taskMetadata),
-		Labels:      GetDefaultLabels(taskMetadata),
-		Job:         job,
+		ClusterName:          cn,
+		Namespace:            taskMetadata.GetNamespace(),
+		Annotations:          GetDefaultAnnotations(taskMetadata),
+		Labels:               GetDefaultLabels(taskMetadata),
+		EnvironmentVariables: GetDefaultEnvVars(taskMetadata),
```
Comment (regadas) [inline pkg/flink/handler.go:100]:
```suggestion
		EnvironmentVariables: GetEnvironmentVariables(taskMetadata),
```
Sorry if my comment was confusing we can leave like you add before 😄

File: pkg/flink/types.go [hunk 4]
```diff
@@ -63,6 +63,15 @@ func GetDefaultLabels(taskCtx pluginsCore.TaskExecutionMetadata) Labels {
 	)
 }
 
+type EnvVars map[string]string
+
+func GetDefaultEnvVars(taskCtx pluginsCore.TaskExecutionMetadata) EnvVars {
```
Comment (regadas) [inline pkg/flink/types.go:68]:
```suggestion
func GetEnvironmentVariables(taskCtx pluginsCore.TaskExecutionMetadata) EnvVars {
```

File: pkg/flink/handler.go [hunk 5]
```diff
@@ -92,11 +93,12 @@ func NewFlinkTaskContext(ctx context.Context, taskCtx FlinkTaskExecContext) (*Fl
...
-		ClusterName: cn,
-		Namespace:   taskMetadata.GetNamespace(),
-		Annotations: GetDefaultAnnotations(taskMetadata),
-		Labels:      GetDefaultLabels(taskMetadata),
-		Job:         job,
+		ClusterName:          cn,
+		Namespace:            taskMetadata.GetNamespace(),
+		Annotations:          GetDefaultAnnotations(taskMetadata),
+		Labels:               GetDefaultLabels(taskMetadata),
+		EnvironmentVariables: GetEnvironmentVariables(taskMetadata),
```
Comment (regadas) [inline pkg/flink/handler.go:100]:
```suggestion
		EnvironmentVariables: GetDefaultEnvironmentVariables(taskMetadata),
```

File: pkg/flink/types.go [hunk 6]
```diff
@@ -63,6 +63,15 @@ func GetDefaultLabels(taskCtx pluginsCore.TaskExecutionMetadata) Labels {
 	)
 }
 
+type EnvVars map[string]string
+
+func GetEnvironmentVariables(taskCtx pluginsCore.TaskExecutionMetadata) EnvVars {
```
Comment (regadas) [inline pkg/flink/types.go:68]:
```suggestion
func GetDefaultEnvironmentVariables(taskCtx pluginsCore.TaskExecutionMetadata) EnvVars {
```

File: pkg/flink/handler.go (hunk 3 above)
Comment (andresgomezfrr) [reply pkg/flink/handler.go]: Oh yeah... I misunderstood it.. sorry! 🙈
